fdbclient/VersionedMap.h

```diff
@@ -425,8 +425,8 @@ namespace PTreeImpl {
 		if (!left) return right;
 		if (!right) return left;
 
-		Reference<PTree<T>> r = Reference<PTree<T>>(new PTree<T>(lastNode(left, at)->data, at));
-		if (EXPENSIVE_VALIDATION) {
+	    Reference<PTree<T>> r = makeReference<PTree<T>>(lastNode(left, at)->data, at);
```

Comment:
I know this is the result of `git clang-format`, but the formatting ends up a little weird here relative to the rest of the file. I'd like to resolve this by applying clang-format to this whole file in a follow up PR. Does that make sense?

Reply:
I think applying `clang-format` to the whole file in a future PR makes sense.